src/default-xpring-client.ts

```diff
@@ -174,6 +176,24 @@ class DefaultXpringClient implements XpringClientDecorator {
 
     return new GetTxResponseWrapper(getTxResponse)
   }
+
+  // TODO(keefertaylor): Add tests for this method once send is hooked up. 
+  private async getFee(): Promise<XRPDropsAmount> {
+    const getFeeRequest = new GetFeeRequest()
+
+    const getFeeResponse = await this.networkClient.getFee(getFeeRequest)
+    const fee = getFeeResponse.getDrops()
+    if (!fee) {
+      throw new Error(XpringClientErrorMessages.malformedResponse)
+    }
+
+    const minimumFee = fee.getMinimumFee()
+    if (!minimumFee) {
+      throw new Error(XpringClientErrorMessages.malformedResponse)
+    }
```

Comment:
if you wanted to, you could do

```
class MalformedResponseError extends Error {}
```

then you can get tighter typing on the error that was thrown, and you could put the error message in there

Reply:
https://app.asana.com/0/1143722888069470/1158635908525161